sklearn/model_selection/_validation.py

```diff
@@ -781,6 +781,27 @@ def _score(estimator, X_test, y_test, scorer, error_score="raise"):
                 UserWarning,
             )
 
+    # Check errors in `_MultimetricScorer`
+    if isinstance(scorer, _MultimetricScorer):
+        exceptions = [
+            (name, e) for name, e in scores.items() if isinstance(e, Exception)
+        ]
+        if exceptions:
+            if error_score == "raise":
+                raise exceptions[0][1]
+            else:
+                for name, e in exceptions:
+                    scores[name] = error_score
+                    details = "".join(
+                        format_exception(etype=type(e), value=e, tb=e.__traceback__)
+                    )
```

Comment:
I chose `format_exception` here, which using this way with `join` gives the same output as `format_exc()`, but now it uses the `e` exception istead of the exception blocks exception.